cms/envs/common.py

```diff
@@ -812,13 +821,24 @@
     'common/js/vendor/require.js'
 ]
 
+capa_js = (
+    rooted_glob(COMMON_ROOT / 'static', 'common/js/javascript_loader.js') +
+    rooted_glob(COMMON_ROOT / 'static', 'common/js/collapsible.js') +
+    rooted_glob(COMMON_ROOT / 'static', 'common/js/capa/*.js') +
+    rooted_glob(COMMON_ROOT / 'static', 'common/js/capa/studio/*.js')
+)
+
 # test_order: Determines the position of this chunk of javascript on
 # the jasmine test page
 PIPELINE_JS = {
     'base_vendor': {
         'source_filenames': base_vendor_js,
         'output_filename': 'js/cms-base-vendor.js',
     },
+    'capa': {
+        'source_filenames': capa_js,
+        'output_filename': 'js/capa.js',
+    },
```

Comment:
Is it necessary to add static assets to the top level config, as opposed to doing it at the XBlock level? Or would that be part of a later move?